target/linux/mediatek/patches-6.1/100-dts-update-mt7622-rfb1.patch

```diff
@@ -35,8 +35,9 @@
  		};
  	};
  
- 	memory {
+-	memory@40000000 {
 -		reg = <0 0x40000000 0 0x20000000>;
++	memory {
```

Comment:
Just keep `memory@40000000`. ref:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit?h=v6.1.68&id=c2adea319187cb6d7e27cacd9aba24f980b7c364

Reply:
Please review my new force pushed commit.

Reply:
Hi! We need to keep the RAM size change. So we just need to edit the original patch to change `memory` to `memory@40000000`.

From
```
 	};

 	memory {
-		reg = <0 0x40000000 0 0x20000000>;
+		reg = <0 0x40000000 0 0x40000000>;
 	};

 	reg_1p8v: regulator-1p8v {
```
to
```
 	};

 	memory@40000000 {
-		reg = <0 0x40000000 0 0x20000000>;
+		reg = <0 0x40000000 0 0x40000000>;
 	};

 	reg_1p8v: regulator-1p8v {
```

Reply:
I see now... please review my new force pushed commit.